src/lib/components/Search.svelte

```diff
@@ -4,20 +4,38 @@
 	export let filter: string = '';
 	export let placeholder = 'Filter...';
 	export let onChange: (value: string) => void;
+	export let searchIcon = false;
+	export let searchInputClass: string = '';
+	export let formClass: string = '';
 
 	function internalOnChange(e: Event & { currentTarget: HTMLInputElement }) {
 		onChange(e.currentTarget.value);
 	}
 </script>
 
-<form on:submit|preventDefault class="mt-2">
-	<div class="input-group">
+<form on:submit|preventDefault class={`${formClass || ''}`}>
+	<div class={`input-group ${searchIcon ? 'search-icon' : ''}`}>
+		{#if searchIcon}
+			<i class="bi bi-search" />
+		{/if}
 		<input
 			type="text"
-			class="form-control"
+			class={`form-control ${searchInputClass || ''}`}
```

Comment:
same here

Reply:
how? `searchInputClass` can be any class given class string.